crates/nu-command/src/generators/seq.rs

```diff
@@ -94,19 +76,9 @@ impl Command for Seq {
                 }),
             },
             Example {
-                description: "sequence 1 to 10 with pipe separator",
-                example: "seq -s '|' 1 10",
-                result: Some(Value::test_string("1|2|3|4|5|6|7|8|9|10")),
-            },
-            Example {
-                description: "sequence 1 to 10 with pipe separator padded with 0",
-                example: "seq -s '|' -w 1 10",
-                result: Some(Value::test_string("01|02|03|04|05|06|07|08|09|10")),
-            },
-            Example {
-                description: "sequence 1 to 10 with pipe separator padded by 2s",
-                example: "seq -s ' | ' -w 1 2 10",
-                result: Some(Value::test_string("01 | 03 | 05 | 07 | 09")),
```

Comment:
it would be nice to keep an example of how to do this since it's quite a bit more difficult now.
```
seq 1 10 | into string | str lpad --character 0 --length 2 | str join ' | '
```

Reply:
No clue what that means, but my guess is that our testing facility is inadequate. I don't think all commands can be tested in examples maybe?

Reply:
Yeah that sound's like the example test runner just having the command under test and some base commands